### PR TITLE
Feature cart page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Cart Page with 
- Cards 
- Price details

Here is the preview link - https://deploy-preview-3--angry-mahavira-118276.netlify.app/

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
+/*Colors not finalised, will use variables later*/
+.cart-card{
+    width: 700px;
+    height: 390px;
```
Comment (Namrata-J) [inline css/cart.css:4]: you can use rem instead

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    right: 5vw;
+    top: 1rem;
+}
+.checkout{
+    margin-left: 1rem;
+    margin-top: 5rem;
+    border: 1px solid black;
+    background-color: cornflowerblue;
+    width: 480px;
+    height: 440px;
```
Comment (Namrata-J) [inline css/cart.css:106]: here as well

**Review by Namrata-J — APPROVED** [review]

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+.cart-card-image{
+    width: 40%;
+    height: 100%;
+}
+.cart-card-image img{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
```
Comment (ArjunGTX) [inline css/cart.css:24]: You can add `object-postion: center` to center the image

File: css/cart.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+}
+
+.cart-card-heading{
+    font-size: 1.5rem;
+    margin-bottom: 0rem;
```
Comment (ArjunGTX) [inline css/cart.css:38]: `0` is enough `rem` is redundant

**Review by ArjunGTX — APPROVED** [review]

File: css/cart.css (hunk 3 above)
Comment (abhijitdotsharma) [reply css/cart.css:24]: I tried it, didn't notice any difference.

File: css/cart.css (hunk 4 above)
Comment (abhijitdotsharma) [reply css/cart.css:38]: Did this, thanks

File: css/cart.css (hunk 2 above)
Comment (abhijitdotsharma) [reply css/cart.css:106]: I use rems for things that might change when I change the base font size, rest of the times px makes sense.

**Pull request merged**